### DISCUSSION TITLE
UCP integration in WRF

### DISCUSSION THREAD

**Discussion by magalietecher** [orbisgis/geoclimate#814] [Q&A]
Hi everybody, 

I'm a PhD student and novice in Geoclimate and WRF simulation. I'm working on the simulation of UHI effects in France.

Maybe I did not find an answer in the forum but I would like to integrate UCP generated by Geoclimate into WRF but I don't know where to start. Do I need to use LCZ simulation integration possible with WRF 4.3?  Or is it a way to integrate UCP directly? Do you have some tutorial? 

Thank you for your suggestions and help!

**Comment by j3r3m1** [comment]
Hi @magalietecher and thank you for your message. I think that a beginning of answer has been provided in this issue. I propose that we only the other issue until there is a more formal tutorial in the GeoClimate wiki, what do you think ?

**Comment by magalietecher** [comment]
Yes thank you! I will try the method proposed in this discussion'
